src/library_glemu.js

```diff
@@ -2088,6 +2096,13 @@ var LibraryGLEmulation = {
               vsFogVaryingInit = '  v_fogFragCoord = abs(ecPosition.z);\n';
             }
 
+            var vsPointSizeDefs = null;
+            var vsPointSizeInit = null;
```

Comment:
should these be `''`s (empty strings) and not `null`?  otherwise when doing the `join` later down we would get lines with `null` on them.

Reply:
This is a copy-paste of the fog initialization code from a few lines above the fold. Didn't want to stand out from the surrounding code, so I didn't change it.

Reply:
Oh, good point, yeah...

Also I checked now and I was wrong, it doesn't convert the inputs to strings first... JS actually has a special-casing of null and undefined and a few others: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join#Description